src/datanode/src/instance.rs

```diff
@@ -330,6 +343,14 @@ impl Instance {
         self.procedure_manager
             .start()
             .context(StartProcedureManagerSnafu)?;
+        #[cfg(feature = "version-report")]
+        {
+            if let Some(task) = &self.version_reporter_task {
+                let _ = task.start(common_runtime::bg_runtime()).map_err(|_e| {
+                    warn!("start version report task error");
```

Comment:
nit: looks a bit confusing the `warn!` level but the message is about `error` 😅

Reply:
Here's just a statistic. Trying to be imperceptible to the user. Probably better not to output logs.I'm not sure.

Reply:
I guess, having this log line is useful, how about rephrasing it?
```
warn!("failed to start version report task");
```
I don't have any strong opinion, feel free to leave it as is